plyer/platforms/linux/wifi.py

```diff
@@ -16,7 +16,35 @@
 
 
 class LinuxWifi(Wifi):
-    names = {}
+    def __init__(self):
+        self.names = {}
+        self.ifname = self._getIfname()
+        self.nmcliVersion = self._getNmcliVersion()
+
+    def _getIfname(self):
+        '''
+        Return wifi interface name if available else return "wlan0'.
+        '''
+        try:
+            com = Popen(["nmcli", "-t", "device"], stdout=PIPE)
+            com = com.communicate()[0].decode().split("\n")
+            for i in com:
+                if "wifi" in i:
+                    return i.split(":")[0]
+            return 'wlan0'
+        except Exception:
```

Comment:
Unless you are actually doing something with the exception, such as letting the user know what and why failed, the effect it the same as before.